resource/secrets.go

```diff
@@ -0,0 +1,53 @@
+package resource
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/open-horizon/anax/containermessage"
+	"github.com/open-horizon/anax/persistence"
+	"path"
+)
+
+func (a AuthenticationManager) HandleNewSerivceSecrets(depDesc *containermessage.DeploymentDescription, agId string) error {
```

Comment:
So you are conflating persistence of the secret(s) with creating them in the file system in this function. I was envisioning that we'd persist the secrets around the same point where the agreement is persisted, and that container.go would come to this new component to retrieve the secrets and put them into the filesystem. I'm thinking about a case where an agreement proposal is followed closely by a secret update. The secret update should be able to find the secret to be updated in the DB, but it might not be there if its not persisted until container.go.

Reply:
Done